thorlcr/activities/funnel/thfunnelslave.cpp

```diff
@@ -85,6 +85,7 @@ class CParallelFunnel : implements IRowStream, public CSimpleInterface
                 inputStream = funnel.activity.queryInputStream(inputIndex);
                 while (!stopping)
                 {
+                    LookAheadTimer timer(funnel.activity.getActivityTimerAccumulator(), funnel.activity.queryTimeActivities());
```

Comment:
not sure about what this is collecting, but think:
- it should include the time it spend in startInput 
- should not include the time it may block in pushMulti

basically, lookahead time should only be the time it spend reading the the input ahead (which in a single thread regular act. would be done inline and be part of total cycles.